src/Database/DBStructure.php

```diff
@@ -1064,7 +1064,7 @@ function ($to, $from) {
 	 * @return boolean Does the table exist?
 	 * @throws Exception
 	 */
-	public static function existsColumn($table, $columns = [])
+	public static function existsColumn(string $table, array $columns = []): bool
```

Comment:
Please update the documentation to reflect the type change with the optional schema prefix.